### PR TITLE
{Release} Upgrade to Azure CLI 2.41.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Azure CLI 2.41.0 release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,142 @@
...
+
+* [BREAKING CHANGE] `az acs`: Remove the deprecated command group (#23784)
+
+**AD**
+
+* Support special characters in user principal name (#23819)
+
+**AKS**
+
+* Fix #23779: `az aks install-cli` support to determine the arch of binaries based on system information (#24006)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:19]:
```suggestion
* Fix #23779: `az aks install-cli`: Support determining the arch of binaries based on system information (#24006)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,142 @@
...
+
+**Backup**
+
+* `az backup policy`: Add support for Smart Tiering policy (#23694)
+
+**Compute**
+
+* [BREAKING CHANGE] `az vmss create`: Update NAT pool to NAT rule V2 for Standard LB SKU when creating VMSS (#23638)
+* `az vm/vmss create`: `--enable-secure-boot` is set to True by default when the `--security-type` used by the VM/VMSS creation is `TrustedLaunch ` (#24004)
+* `az restore-point create`: Add new parameters `--consistency-mode` to support setting consistency mode (#23789)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:54]:
```suggestion
* `az restore-point create`: Add new parameter `--consistency-mode` to support setting consistency mode (#23789)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,142 @@
...
+* `az vmss create/update`: Add new parameters `--priority-count` and `--priority-percentage` to support setting priority mix policy (#23786)
+* `az vm/vmss create/update`: Add new parameters `--disk-controller-type` to support setting disk controller type (#23710)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:56]:
```suggestion
* `az vm/vmss create/update`: Add new parameter `--disk-controller-type` to support setting disk controller type (#23710)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,142 @@
...
+**Storage**
+
+* `az storage blob/container`: Support `--account-name` for non-standard account URL (#23832)
+* `az storage account update`: Fix updating `--default-share-permission` will wipe out other ADProperties issue (#23986)
+* Fix #19311: `az storage remove`: Add support for connection-string (#24049)
+
+**Synapse**
+
+* Update Synapse contacts (#23765)
+
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:141]:
```suggestion
```

Comment (wangzelin007) [inline src/azure-cli/HISTORY.rst:141]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,142 @@
...
+* `az mysql flexible-server update`: Expose `--geo-redundant-backup` argument (#23871)
+* `az mysql/postgres flexible-server create/update`: Deprecate `Enabled` for `--high-availability` argument (#23847)
+* `az mysql flexible-server stop`: Change stopped time logging message (#23979)
+* `az mysql flexible-server ad-admin delete`: Disable `aad_auth_only` when dropping AAD admin (#24002)
+* `az mysql flexible-server identity remove`: Allow removing all identities in a MySQL server (#24042)
+
+**Reservations**
+
+* Remove commands from azure-cli (#24097)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:116]:
```suggestion
* Move commands from azure-cli to azure-cli-extensions (#24097)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,142 @@
...
+* `az spring-cloud connection create postgres`: Add `--system-identity` for springcloud-postgres connection (#22459)
+
+**SQL**
+
+* `az sql server audit-policy show`: Add `isManagedIdentityInUse` info in output (#23275)
+
+**Storage**
+
+* `az storage blob/container`: Support `--account-name` for non-standard account URL (#23832)
+* `az storage account update`: Fix updating `--default-share-permission` will wipe out other ADProperties issue (#23986)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:135]:
```suggestion
* `az storage account update`: Fix ADProperties wipe out issue when updating `--default-share-permission` (#23986)
```

**Review by wangzelin007 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,138 @@
...
+* `az deployment group what-if`: Fix an issue where `complete` deployment mode does not work (#23941)
+
+**Backup**
+
+* `az backup policy`: Add support for Smart Tiering policy (#23694)
+
+**Compute**
+
+* [BREAKING CHANGE] `az vmss create`: Update NAT pool to NAT rule V2 for Standard LB SKU when creating VMSS (#23638)
+* `az vm/vmss create`: `--enable-secure-boot` is set to True by default when the `--security-type` used by the VM/VMSS creation is `TrustedLaunch ` (#24004)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:53]:
```suggestion
* `az vm/vmss create`: `--enable-secure-boot` is set to True by default when the `--security-type` used by the VM/VMSS creation is `TrustedLaunch` (#24004)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,138 @@
...
+
+* `az mysql flexible-server update`: Expose `--geo-redundant-backup` argument (#23871)
+* `az mysql/postgres flexible-server create/update`: Deprecate `Enabled` for `--high-availability` argument (#23847)
+* `az mysql flexible-server stop`: Change stopped time logging message (#23979)
+* `az mysql flexible-server ad-admin delete`: Disable `aad_auth_only` when dropping AAD admin (#24002)
+* `az mysql flexible-server identity remove`: Allow removing all identities in a MySQL server (#24042)
+
+**Reservations**
+
+* Move commands from azure-cli to azure-cli-extensions (#24097)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:116]:
```suggestion
* Move commands from azure-cli to `reservation` extension (#24097)
```

**Review by kairu-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Pull request merged**